share/commands/agent-policy-server

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# @description displays or set the policy server
+# @man If called without arguments, displays current policy server.
+# @man Sets the epolicy server to the hostname or IP given.
+# @man +
+# @man *Arguments*:
+# @man +
+# @man *server*: hostname or IP of the policy server to set
```

Comment:
what if the first argument is -h ?

Reply:
why would it be?

Reply:
because all other command have option parsing, but ok -h is already handled by the parent command

Shouldn't we check that the root server name is valid ? resolvable ?

Reply:
It may not be resolvable at the moment it is configured (provisionning, etc) so I'm not sure